project/src/main/puzzle/puzzle-score.gd

```diff
@@ -26,9 +26,11 @@ signal after_game_ended
 
 signal level_index_changed(value)
 
+signal added_line_score(combo_score, box_score)
 signal score_changed
 
 signal combo_ended
+signal topped_out
```

Comment:
Preserve `# emitted when the current piece can't be placed in the _playfield` comment

Reply:
Kept comment